core/src/components/textarea/textarea.scss

```diff
@@ -330,12 +330,8 @@
 // ----------------------------------------------------------------
 
 .native-wrapper {
-  display: flex;
-
   position: relative;
 
-  flex-grow: 1;
```

Comment:
These weren't doing anything and were likely a copy-paste goof while adding the modern syntax; `.native-wrapper` is set to `display: grid` a few lines below this.